Assets/HoloToolkit/Input/Tests/Prefabs/FocusedObjectSphere.prefab

```diff
@@ -0,0 +1,117 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
```

Comment:
Agreed, a lot of these seem like they should just be elements of the scene and don't require prefabs.